Project.toml

```diff
@@ -9,18 +9,19 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DoubleFloats = "497a8b3b-efae-58df-a0af-a86822472b78"
 Mmap = "a63ad114-7e13-5084-954f-fe012c677804"
 Nullables = "4d1e1d77-625e-5b40-9113-a560ec7a8ecd"
-PooledArrays = "2dfb63ee-cc39-5dd5-95bd-886bf059d720"
 WeakRefStrings = "ea10d353-3f73-51f8-a26c-33c1cb351aa5"
 
 [compat]
-PooledArrays = "0.2, 0.3, 0.4, 0.5"
+CodecZlib = "0.6"
```

Comment:
Previous versions of this package needs to gett these upper bounds as well (requires modifying the bounds in the registry). Otherwise the resolver will just use one of those old versions whenever there is an i compatibility with these bounds.

Reply:
Is there hope that https://github.com/bcbi/RetroCap.jl will just handle this for me?

Reply:
Maybe... Cool, I hadn't seen that.